source/sdk/Text.ooc

```diff
@@ -326,6 +326,10 @@ operator == (left: String, right: Text) -> Bool { Text new(left) == right }
 operator != (left: String, right: Text) -> Bool { !(left == right) }
 operator []= (left: TextBuffer, range: Range, right: Text) { right _buffer copyTo(left slice(range min, range count)) }
 
+extend Bool {
+	toText: func -> Text { this ? t"true" : t"false" }
+}
+
```

Comment:
The reason all `toText` were put in this file was because, at the time, `Text.ooc` resided in `source/base`. Now that `Text` is a part of the sdk, we could put all `toText` functions in `Numbers.ooc` instead, alongside `toString`. What do you think?